spec/biblio.json

```diff
@@ -1,111 +1,2 @@
 [
```

Comment:
Should we just get rid of `biblio.json` at this point? Now that 262 and 402 both have published bibliographies, maybe it's no longer necessary.

Reply:
Sure. Added 9c02c45b154484bbaaa894d2d099ee32f3a3e6de to remove biblio.json.